src/DssSpell.sol

```diff
@@ -74,190 +120,324 @@ contract DssSpellAction is DssAction {
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_PAX).deny(MCD_ESM);
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_PAX).deny(address(this));
 
-        DssExecLib.setChangelogAddress("RWA015_A_OUTPUT_CONDUIT", RWA015_A_OUTPUT_CONDUIT);
-        // Add Legacy Conduit to Changelog
-        DssExecLib.setChangelogAddress("RWA015_A_OUTPUT_CONDUIT_LEGACY", RWA015_A_OUTPUT_CONDUIT_LEGACY);
+        // Revoke permissions on the old RWA015_A_OUTPUT_CONDUIT_USDC
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_USDC).nope(RWA015_A_OPERATOR);
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_USDC).hate(RWA015_A_OPERATOR);
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_USDC).diss(RWA015_A_CUSTODY);
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_USDC).file("quitTo", address(0));
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_USDC).deny(MCD_ESM);
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT_USDC).deny(address(this));
+
+        // Remove Legacy Conduit From Chainlog
+        ChainlogLike(DssExecLib.LOG).removeAddress("RWA015_A_OUTPUT_CONDUIT_LEGACY");
+
+        // ----- Add Cron Jobs to Chainlog -----
+        // Forum: https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292
+
+        DssExecLib.setChangelogAddress("CRON_SEQUENCER",       CRON_SEQUENCER);
+        DssExecLib.setChangelogAddress("CRON_AUTOLINE_JOB",    CRON_AUTOLINE_JOB);
+        DssExecLib.setChangelogAddress("CRON_LERP_JOB",        CRON_LERP_JOB);
+        DssExecLib.setChangelogAddress("CRON_D3M_JOB",         CRON_D3M_JOB);
+        DssExecLib.setChangelogAddress("CRON_CLIPPER_MOM_JOB", CRON_CLIPPER_MOM_JOB);
+        DssExecLib.setChangelogAddress("CRON_ORACLE_JOB",      CRON_ORACLE_JOB);
+
+        // ----- Deploy FlapperUniV2 -----
+        // https://vote.makerdao.com/polling/QmQmxEZp#poll-detail
+        // Forum: https://forum.makerdao.com/t/introduction-of-smart-burn-engine-and-initial-parameters/21201
+        // dss-flappers @ b10f68224c648166cd4f9b09595412bce9824301
+
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
+        FlapperInstance memory flap = FlapperInstance({
+            flapper: MCD_FLAP,
+            mom:     FLAPPER_MOM
+        });
+        FlapperUniV2Config memory cfg = FlapperUniV2Config({
+            hop:  1577 seconds,
+            want: 98 * WAD / 100,
+            pip:  PIP_MKR,
+            hump: 50 * MILLION * RAD,
+            bump: 5 * THOUSAND * RAD
+        });
+
+        FlapperInit.initFlapperUniV2({
+            dss: dss,
+            flapperInstance: flap,
+            cfg: cfg
+        });
+
+        FlapperInit.initDirectOracle({
+            flapper: MCD_FLAP
+        });
+
+        DssExecLib.setChangelogAddress("PIP_MKR", PIP_MKR);
+
+        DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_FLAP_JOB);
+        DssExecLib.setChangelogAddress("CRON_FLAP_JOB", CRON_FLAP_JOB);
```

Comment:
The rationale for separating these in the sheet and the copy was because the provenance is different, `FlapJob` being added is part of the Smart Burn Engine deployment while the others were added as a housekeeping addition.

Reply:
This is fine then